### ISSUE TITLE
PyTorch Implmentation of triplet loss

### ISSUE THREAD

**Issue by KinWaiCheuk** [omoindrot/tensorflow-triplet-loss#56]
Hi Olivier, I contacted you almost two years ago regarding the PyTorch implmentation. Now I have finished rewritting your code in PyTorch.
https://github.com/KinWaiCheuk/pytorch-triplet-loss

Would you like me to have a pull request so that your code contains both tensorflow and pytorch implmentation?
Sorry about that I didn't not fork your repo two years ago, I was still a noob in github at that time.
